scripts/eject.js

```diff
@@ -75,6 +75,8 @@ prompt(
       .replace(/^\/\*\*(\*(?!\/)|[^*])*\*\//, '')
       // Remove license header from AppleScript
       .replace(/^--.*\n/gm, '')
+      // Remove dead code on eject
+      .replace(/\/\/ @remove-on-eject-begin([\s\S]*?)\/\/ @remove-on-eject-end/g, '')
```

Comment:
@gaearon ,

I want to double check your question.

The current RegExp `/^\/\*\*(\*(?!\/)|[^*])*\*\//` removes all multiline JS comments, which could contains useful information or other License.

We have 2 solutions for this problem

#### option 1
a.  Remove those 4 lines in `eject.js`

```
// Remove license header from JS
.replace(/^\/\*\*(\*(?!\/)|[^*])*\*\//, '')
// Remove license header from AppleScript
.replace(/^--.*\n/gm, '')
```
b. add a RegExp for using `@remove-on-eject-begin/end` in AppleScript.

c. add `@remove-on-eject-begin/end` before/after the license statement of each file.

#### option 2
Update the RegExp so that it only matches Facebook license header in JS and AppleScript


I prefer **option 1**, which gives us the potential of retaining license in some files (e.g util scripts, 3rd party code).